src/lib.rs

```diff
@@ -373,7 +373,7 @@ mod primitive_date_time;
 mod rand;
 #[cfg(serde)]
 #[allow(missing_copy_implementations, missing_debug_implementations)]
-mod serde;
+pub mod serde;
```

Comment:
This'll likely cause merging/rebasing issues for v0.3, since the mod was removed, but that's nothing to worry about. I'll just handle the conflict when it arises.

Note to self: be sure to remove the `#[allow]` for v0.3.